projects/ids-enterprise-ng/src/lib/rating/soho-rating.component.ts

```diff
@@ -52,4 +52,19 @@ export class SohoRatingComponent implements AfterViewInit, OnDestroy {
       this.rating.enable();
     }
   }
+
+  /** Gets the current value of the control. If the control doesn't exist, will return -1 */
+  public get currentValue() {
```

Comment:
Do we want to call it val() in both places? I actually prefer currentValue so ok with leaving this for the public API but most of the time we stick to the name...